tests/components/hue/test_init_multiple_bridges.py

```diff
@@ -0,0 +1,185 @@
+"""Test Hue init with multiple bridges."""
+
+from aiohue.groups import Groups
+from aiohue.lights import Lights
+from aiohue.scenes import Scenes
+from aiohue.sensors import Sensors
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components import hue
+from homeassistant.components.hue import sensor_base as hue_sensor_base
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import Mock, patch
+
+
+async def setup_component(hass):
+    """Hue component."""
+    with patch.object(hue, "async_setup_entry", return_value=True):
+        assert (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)